src/stactools/sentinel2/stac.py

```diff
@@ -203,14 +204,16 @@ def create_item(
         )
 
     # View Extension
+    view = ViewExtension.ext(item, add_if_missing=True)
+    view.azimuth = mean([v.azimuth for b, v in metadata.viewing_angles.items()])
+    view.incidence_angle = mean([v.zenith for b, v in metadata.viewing_angles.items()])
```

Comment:
```suggestion
    view.incidence_angle = mean([v.zenith for b, v in metadata.viewing_angles.values()])
```

`view.incidence_angle = mean(v.zenith for v in metadata.viewing_angles.values())`
might also work if mean takes an interable